scripts/prepack.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+set -o pipefail
+
+if [[ -n $SKIP_PREPACK ]]; then
+  echo "Notice: skipping prepack."
+  exit 0
+fi
+
+yarn build
```

Comment:
This should be `yarn build:clean` so that we make sure to have a clean dist output